content/en/docs/concepts/policy/limit-range.md

```diff
@@ -26,7 +26,7 @@ A _LimitRange_ provides constraints that can:
 
 ## Enabling LimitRange
 
-LimitRange support is enabled by default for many Kubernetes distributions. It is
+LimitRange support is enabled by default since Kubernetes 1.10. It can also be explicitly
 enabled when the apiserver `--enable-admission-plugins=` flag has `LimitRanger` admission controller as
 one of its arguments.
```

Comment:
I'm not sure, on reading this, what to deduce.
- LimitRange is GA. `kube-apiserver` has LimitRange resources built in to it. If you specify `--enable-admission-plugins=` and don't specify the `LimitRanger` admission controller in the list, your cluster will not implement working LimitRanges. If you don't set `--enable-admission-plugins=` at all the admission controller is automatically selected.
- LimitRange is GA. `kube-apiserver` has LimitRange resources built in to it. If you specify `--enable-admission-plugins=` and don't specify the `LimitRanger` admission controller in the list, your cluster will not implement working LimitRanges. If you don't set `--enable-admission-plugins=` at all the admission controller is not selected and LimitRanges are not enforced.
- something else

Reply:
@sftim That makes sense. 

I've simplified the doc to one sentence. 
> LimitRange support has been enabled by default since Kubernetes 1.10.

Let me know what you think.